opencga-core/src/main/java/org/opencb/opencga/core/models/JwtPayload.java

```diff
@@ -0,0 +1,118 @@
+package org.opencb.opencga.core.models;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.commons.lang3.StringUtils;
+import org.opencb.commons.datastore.core.ObjectMap;
+import org.opencb.opencga.core.common.JacksonUtils;
+
+import java.util.Base64;
+import java.util.Date;
+
+public class JwtPayload {
+
+    private String userId;
```

Comment:
Shouldn't these fields be "final"?

Reply:
True, to be changed in the next release.